### PR TITLE
Change pep8 to flake8 and fix linter errors

### PR DESCRIPTION
**Pull request by fulaphex**
Implementation of #280 
`# flake8: noqa` at the top of the `__init__.py` files to get rid of the warnings about unused imports

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,9 @@ stages:
   - name: deploy
     # require any tag name to deploy
     if: tag =~ .*
+before_install:
+  - sudo apt-get update
+  - sudo apt-get install -y libsnappy-dev
```
Comment (vmarkovtsev) [inline .travis.yml]: Better to use apt add-ons - we could run it on macos too

**Comment by vmarkovtsev** [comment]
It is a pity that we require `apt-get` though, it makes the builds longer and more unstable. Roughly every 10th build used to fail in the past. @fulaphex do we really have to install snappy?

**Comment by fulaphex** [comment]
@vmarkovtsev the installation of `python-snappy` was failing on my fork when running on Travis. [travis log](https://travis-ci.org/fulaphex/ml/jobs/395473182)

It's no longer an issue ([new log](https://travis-ci.org/fulaphex/ml/jobs/399152537)), so I'll remove and PR the change.

**Comment by vmarkovtsev** [comment]
@fulaphex I see, it is required by parquet. The reason it did not fail is because the cache was used. So nothing can be done, alas!

**Pull request merged**